api-datos/cmd/tablacruce/modelos.go

```diff
@@ -23,6 +23,11 @@ type Island struct {
 	Latitud  float64
 }
 
+// NewIsland Crea una nueva estructura de tipo Island
+func NewIsland(id int, nombre string, longitud, latitud float64) Island {
```

Comment:
Parece que falta especificar el tipo del argumento de entrada:
```suggestion
func NewIsland(id int, nombre string, longitud float64, latitud float64) Island {
```